specification.md

```diff
@@ -3271,8 +3271,8 @@ defined via the `parameters` property in [function definitions](#FunctionRef-Def
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
 | name | Unique event name | string | yes |
-| source | CloudEvent source | string | yes if kind is set to "consumed", otherwise no |
-| type | CloudEvent type | string | yes |
+| source | CloudEvent source | string | no if `type` is set, otherwise yes. If not set when `kind` is `produced`, runtimes are expected to use a default value, such as https://serverlessworkflow.io in order to comply with the [CloudEvent spec constraints](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#source-1) |
```

Comment:
I'd say that "runtimes are expected to validate the incoming and outgoing CE to comply with CloudEvent spec constraints...". Not here, but in a paragraph after this table since this sentence suggests overall validation of CEs.